Documentation/ceph-block.md

```diff
@@ -42,20 +44,24 @@ apiVersion: storage.k8s.io/v1
 kind: StorageClass
 metadata:
    name: rook-ceph-block
-provisioner: ceph.rook.io/block
+provisioner: rbd.csi.ceph.com
 parameters:
-  blockPool: replicapool
-  # The value of "clusterNamespace" MUST be the same as the one in which your rook cluster exist
-  clusterNamespace: rook-ceph
-  # Specify the filesystem type of the volume. If not specified, it will use `ext4`.
-  fstype: xfs
-# Optional, default reclaimPolicy is "Delete". Other options are: "Retain", "Recycle" as documented in https://kubernetes.io/docs/concepts/storage/storage-classes/
-reclaimPolicy: Retain
+    # clusterID is the namespace where the rook cluster is running
+    clusterID: rook-ceph
```

Comment:
Eventhough this can be any string it should be a unique identifier of the cluster. Atleast I used to put `fs id` in this field. Not sure `rook-ceph` can cause confusion that , people think its the namespace name :)